ClientRequirments.md

```diff
@@ -0,0 +1,39 @@
+# Platform Requirements for Fluid Framework Client Packages
+
+These are the platform requirements for the current version of Fluid Framework Client Packages.
+These requirements error on the side of being too strict since within a major version they can be relaxed over time, but not made stricter.
```

Comment:
```suggestion
These requirements err on the side of being too strict since within a major version they can be relaxed over time, but not made stricter.
```